adapter/src/test/java/com/twilio/kudu/sql/PaginationIT.java

```diff
@@ -449,11 +481,11 @@ public void testPaginationMultipleAccountsOrderedByTimestamp() throws Exception
       TimestampString lowerBoundDateInitiated = TimestampString.fromMillisSinceEpoch(T1);
       TimestampString upperBoundDateInitiated = TimestampString.fromMillisSinceEpoch(T4);
       String dateInitiatedOrder = descending ? "DESC" : "ASC";
-      String firstBatchSqlFormat = "SELECT * FROM %s WHERE (account_sid = '%s' OR account_sid = "
+      String firstBatchSqlFormat = "SELECT * FROM %s %s WHERE (account_sid = '%s' OR account_sid = "
           + "'%s') AND date_initiated >= TIMESTAMP'%s' AND date_initiated < TIMESTAMP'%s' "
           + "ORDER BY date_initiated %s, transaction_id " + "LIMIT 7";
-      String firstBatchSql = String.format(firstBatchSqlFormat, tableName, ACCOUNT1, ACCOUNT2, lowerBoundDateInitiated,
-          upperBoundDateInitiated, dateInitiatedOrder);
+      String firstBatchSql = String.format(firstBatchSqlFormat, tableName, hint, ACCOUNT1, ACCOUNT2,
```

Comment:
Curious how this sql looks ? 
There is a hint immediately after the table name which can be "DISABLE_IN_LIST_OPTIMIZATION" ?

Reply:
sql looks like this `SELECT * FROM TABLE_ASC /*+ DISABLE_IN_LIST_OPTIMIZATION */ WHERE (account_sid = 'ACCOUNT1' OR account_sid = 'ACCOUNT2') AND date_initiated >= TIMESTAMP'1970-01-01 00:00:01' AND date_initiated < TIMESTAMP'1970-01-01 00:00:04' ORDER BY date_initiated ASC, transaction_id LIMIT 7`

https://docs.cloudera.com/documentation/enterprise/6/6.3/topics/impala_hints.html